js/settingsHelpers.js

```diff
@@ -361,11 +361,11 @@ export const changeName = async (firstName, lastName, middleName, suffix, prefer
     [cId.lName]: lastName,
     [cId.suffix]: parseInt(suffix),
     [cId.prefName]: preferredFirstName,
-    ['query.firstName']: firstName.toLowerCase(),
-    ['query.lastName']: lastName.toLowerCase(),
   };
-
-  const { changedUserDataForProfile, changedUserDataForHistory } = findChangedUserDataValues(newValues, userData, 'changeName');
+  
+  let { changedUserDataForProfile, changedUserDataForHistory } = findChangedUserDataValues(newValues, userData, 'changeName');
+  changedUserDataForProfile = handleNameField(firstNameTypes, 'firstName', changedUserDataForProfile, userData);
+  changedUserDataForProfile = handleNameField(lastNameTypes, 'lastName', changedUserDataForProfile, userData);
   const isSuccess = processUserDataUpdate(changedUserDataForProfile, changedUserDataForHistory, userData[cId.userProfileHistory], userData[cId.prefEmail], 'changeName');
   return isSuccess;
```

Comment:
Agreed, it could be assigned directly. I would like to avoid this update at this time since it was not edited in this PR and we're on a tight timeframe with testing.